bodhi/cli.py

```diff
@@ -149,16 +151,45 @@ def comment(update, text, karma, user, password, **kwargs):
 
 
 @updates.command()
-@click.argument('update')
-@click.option('--user', envvar='USERNAME')
-@click.option('--password', prompt=True, hide_input=True)
 @click.option('--staging', help='Use the staging bodhi instance',
               is_flag=True, default=False)
-def download(update, **kwargs):
-    client = Bodhi2Client(username=user, password=password, staging=kwargs['staging'])
-    print('%r %r %r' % (update, text, karma))
-    resp = client.comment(update, text, karma)
-    print_resp(resp, client)
+@click.option('--cves', help='Download update(s) by CVE(s) (comma-separated list)')
+@click.option('--updateid', help='Download update(s) by ID(s) (comma-separated list)')
+@click.option('--builds', help='Download update(s) by build NVR(s) (comma-separated list)')
+def download(**kwargs):
+    client = Bodhi2Client(staging=kwargs['staging'])
+    del(kwargs['staging'])
+    # At this point we need to have reduced the kwargs dict to only our
+    # query options (cves, updateid, builds)
+    if not any(kwargs.values()):
+        click.echo("ERROR: must specify at least one of --cves, --updateid, --builds")
+        sys.exit(1)
+
+    # As the query method doesn't let us construct OR queries, we're
+    # gonna run one query for each option that was passed. The syntax
+    # for this is a bit ugly, sorry.
+    for item in kwargs.items():
```

Comment:
Maybe this could be rephrased for readability a bit.. like:

```python
for attr, value in kwargs.items():
    if not value:
        continue
    expecteds = len(value.split(','))
    resp = client.query(**{attr: value})
    # etc...
```

Names are nicer than indexing ``0`` and ``1`` everywhere.